src/constants.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef CLP_FFI_JS_CONSTANTS_HPP
+#define CLP_FFI_JS_CONSTANTS_HPP
+
+#include <array>
+
+namespace clp_ffi_js {
+using namespace std::literals::string_view_literals;
```

Comment:
I'd prefer to not use `using namespace` in a header file.
Maybe just explicitly declare the array with type and length?

Reply:
Do we have to specify the strings as string views? If we are fine with the strings being c-strings, we also don't have to add the template specifiers. e.g.,
```
constexpr std::array cLogLevelNames{
        "NONE",  // This should not be used.
        "TRACE",
        "DEBUG",
        "INFO",
        "WARN",
        "ERROR",
        "FATAL",
};
```

Reply:
it would be better to be explicit using string view. What's your concern of using `std::string_view` directly in the array template?

Reply:
(was too lazy to count to 7, lol.
I did find the compiler errors out when the given length is too short, say 6, but not when the length is too long, say 8.

it's updated now